app/controllers/subscriptions_controller.rb

```diff
@@ -1,9 +1,9 @@
 class SubscriptionsController < ApplicationController
   include FrequenciesHelper
   before_action :assign_attributes
-  before_action :assign_back_url
 
   def new
+    assign_back_url
```

Comment:
This is better than doing it everywhere! I'm finding the change a bit confusing, though, as here we're calling `assign_back_url`, whereas on line 34, we're assigning `@back_url` manually. Looking at the `assign_back_url`, I can see that each line is specific to one of the two places where it's called.

For simplicity and consistency, could we remove the `assign_back_url` method altogether?

Reply:
good point. fixed